frontend/packages/console-dynamic-plugin-sdk/README.md

```diff
@@ -164,6 +164,7 @@ This section documents notable changes in the Console provided shared modules ac
   removed in the future. Plugins should provide their own CSS to spin icons if needed.
 - Removed PatternFly 4.x shared modules.
 - Upgraded PatternFly to v6.
```

Comment:
hmm could you also mention in this changelog that PF5 is/will be deprecated when PF7 comes out?

Reply:
We certainly could.  But wonder if we shouldn't plan that as a separate deprecation notice for a future SDK release so it doesn't get overlooked?

Reply:
I'm not sure what the best plan of action would be so that plugins migrate to PF6 faster, I guess a note in a README didn't work before

/lgtm for now, we can address PF5 deprecation later